### PR TITLE
Add a warning widget if API does not support the automatic token revoke

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?
Add a warning widget if API does not support the automatic token revoke.

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/22039

Знімок екрана 2023-04-04 о 20 29 04

### CODE REVIEW

File: packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap [hunk 1]
```diff
@@ -249,6 +249,283 @@ Array [
             onMouseEnter={[Function]}
           >
             GitLab
+            <span
+              data-testid="gitlab-unsupported-automatic-revoke-tooltip"
+              data-tip="Provided API does not support the automatic token revoke. You can revoke it manually."
```
Comment (ibuziuk) [inline packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap:254]: Can we add URL in the tooltip that would be `GitLab` specific?

File: packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap [hunk 2]
```diff
@@ -249,6 +249,283 @@ Array [
...
+          <td
+            className=""
+            data-key={1}
+            data-label="Name"
+            onMouseEnter={[Function]}
+          >
+            Microsoft Azure DevOps
+            <span
+              data-testid="azure-devops-unsupported-automatic-revoke-tooltip"
+              data-tip="Provided API does not support the automatic token revoke. You can revoke it manually."
```
Comment (ibuziuk) [inline packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap:504]: Can we add URL in the tooltip that would be Azure DevOps specific?

**Review by ibuziuk — COMMENTED** [review]
Look great, let's just add specific URL for each provide to make the manual revocation straightforward

File: packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap (hunk 2 above)
Comment (olexii4) [reply packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap]: I have fixed it.

File: packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap [hunk 3]
```diff
@@ -249,6 +249,283 @@ Array [
...
+          <td
+            className=""
+            data-key={1}
+            data-label="Name"
+            onMouseEnter={[Function]}
+          >
+            Bitbucket
+            <span
+              data-testid="bitbucket-unsupported-automatic-revoke-tooltip"
+              data-tip="Provided API does not support the automatic token revoke. You can revoke it manually."
```
[the remark this answers is not in the document]
Comment (olexii4) [reply packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap]: I have fixed it.

File: packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap (hunk 1 above)
Comment (olexii4) [reply packages/dashboard-frontend/src/pages/UserPreferences/GitServicesTab/__tests__/__snapshots__/index.spec.tsx.snap]: I have fixed it.

**Review by akurinnoy — APPROVED** [review]

**Review by ibuziuk — APPROVED** [review]

**Pull request merged**